frontend/packages/data-portal/app/components/Run/RunHeader.tsx

```diff
@@ -51,14 +52,28 @@ export function RunHeader() {
     annotationFilesAggregates,
     tomogramsCount,
     alignmentsCount,
+    tomograms,
   } = useRunById()
   const { toggleDrawer } = useMetadataDrawer()
   const { t } = useI18n()
 
   const tiltSeries = run.tiltseries[0]
 
   const tomogram = run.tomogram_voxel_spacings.at(0)?.tomograms.at(0)
-  const keyPhotoURL = tomogram?.key_photo_url ?? undefined
+
+  // Use author submitted tomogram if available, otherwise default to the first one
+  const tomogramV2 =
+    tomograms.find(
+      (currentTomogram) =>
+        currentTomogram.isVisualizationDefault ??
```

Comment:
things might've changed but I thought they wanted `isAuthorSubmitted` first cuz it has all the annotations or something? or did they make `isVisualizationDefault` the same as `isAuthorSubmitted`...